deepchecks/analytics/utils.py

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils for analytics."""
+import os
+import uuid
+from typing import Dict, Text, Any
+
+import yaml
+
+CONFIG_FILE_PATH = '~/.deepchecks/config.yml'
```

Comment:
wouldn't it be easier to put here the -
CONFIG_FILE_PATHE = os.path.expanduser('~/.deepchecks/config.yml')?